src/app/app.component.ts

```diff
@@ -217,7 +220,7 @@ export class CopayApp {
       // Only overlay for iOS
       if (this.platform.is('ios')) {
         this.statusBar.overlaysWebView(true);
-        this.statusBar.styleDefault();
+        this.themeProvider.useDefaultStatusBar(true);
```

Comment:
`cordova-plugin-statusbar` detect automatically dark mode. I think it's not necessary to make changes. Also the app is forcing that to use "light" mode: https://github.com/bitpay/copay/blob/master/app-template/config-template.xml#L111